plugins/modules/foreman_global_parameter.py

```diff
@@ -43,6 +44,20 @@
     description:
       - Value of the Global Parameter
     required: false
+  parameter_type:
+    description:
+      - Type of value
+    required: true
```

Comment:
I think, we should not require this, but default it to string.
At least for users of <1.22 having to set it seems very weird.

Reply:
Yepp, that's how I'd do it too.